src/en/send.md

```diff
@@ -97,58 +97,100 @@ You can leave out this argument if your service only has one reply-to email addr
 Sending files by email is not enabled by default. To turn on this feature, [contact us](https://notification.canada.ca/contact).
 :::
 
-To send a file by email, you'll need to add a placeholder to the template then upload a file. The placeholder will contain a secure link to download the file. 
+### File types and size requirements
+You can upload PDF, CSV, .jpeg, .png, .odt, .txt, .rtf,  Microsoft Excel and Microsoft Word Document files. Your file must be smaller than 10MB.
+
+[Contact us](https://notification.canada.ca/contact) if you need to send other file types.
+
+
+### Sending methods
+You can send files in two ways on Notify:
+
+1. directly attached to emails
+1. through a unique link to a web interface. Links are unique and unguessable.
+
+You are able to control how files are delivered to recipients on every
+API call. GC Notify cannot access or decrypt your files.
+
+::: tip Choosing the appropriate sending method
 
-The links are unique and unguessable. GC Notify cannot access or decrypt your file.
+People are generally more used to files directly attached to emails. With that in mind, it's not uncommon to see attachments being blocked by security rules or some email providers. Use the unique link method to prevent your attachments from being blocked.
+
+We encourage you to perform tests before choosing a sending method.
+
+:::
 
 ### Add contact details to the file download page
 
 To turn on this feature, [contact us](https://notification.canada.ca/contact).
 
-### Add a placeholder to the template
+### Add a placeholder to the template (optional)
+
+::: tip Only relevant if using unique links
+
+Adding a placeholder in your template is only needed if you’re sending files with a unique link. You do not need to add a placeholder if you’re sending files as direct attachments.
+
```

Comment:
Placeholders are ignored for direct attachments, in addition placeholders for direct attachments put into the template will surface an error (missing personalisation).